tests/sharding/test_vmc_handler.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+
+import rlp
+
+from eth_tester.backends.pyevm.main import get_default_account_keys
+
+import eth_utils
+
+from evm.utils.address import (
+    generate_contract_address,
+)
+
+from evm.chains.sharding.mainchain_handler.config import (
+    PERIOD_LENGTH,
+    SHUFFLING_CYCLE_LENGTH,
+)
+
+from evm.chains.sharding.mainchain_handler.vmc_handler import (
+    VMCHandler,
+)
+
+from evm.chains.sharding.mainchain_handler import (
+    vmc_utils,
+)
+
+from evm.chains.sharding.mainchain_handler.backends.chain_handler import (
+    RPCChainHandler,
+)
+
+from evm.chains.sharding.mainchain_handler.backends.tester_chain_handler import (
+    TesterChainHandler,
+)
+
+keys = get_default_account_keys()
+
+sha3 = eth_utils.crypto.keccak
```

Comment:
Can we remove this alias and just use `keccak`?

Reply:
no problem:)